articles/welcome.md

```diff
@@ -1,36 +1,36 @@
 
-[Why should I learn quantum computing?](xref:microsoft.quantum.overview.why)
+[Why learn quantum computing?](xref:microsoft.quantum.overview.why)
 
-[What is the Q# programming language?](xref:microsoft.quantum.overview.qsharp)
+[What is Q#?](xref:microsoft.quantum.overview.qsharp)
 
-[How can I learn quantum computing?](xref:microsoft.quantum.overview.learn)
+[How to learn quantum computing with Q#?](xref:microsoft.quantum.overview.learn)
 
 ## Tutorials to get you started learning and coding
 
-Next, we provide a short tutorial of how to [write your first Q# program](xref:microsoft.quantum.write-program). If you are ready to start coding, this tutorial guides you on installing the QDK and writing your first Q# program that demonstrates some of the first concepts of quantum programming. If you are not ready to start coding, you can still follow along with the tutorial without installing the QDK and get an overview of the Q# programming language and the first concepts of quantum computing.
+Next, we provide a short Getting Started guide of how to [write your first Q# program](xref:microsoft.quantum.write-program). If you are ready to start coding, this guides you on installing the QDK and writing your first Q# program that demonstrates some of the first concepts of quantum programming. If you are not ready to start coding, you can still follow along without installing the QDK and get an overview of the Q# programming language and the first concepts of quantum computing.
 
-The second tutorial, [Grover’s search](xref:microsoft.quantum.quickstarts.search), offers a nice example of a Q# program that demonstrates the different way of solving real problems with quantum computing.  With this tutorial, you get a sense of how different quantum computing is from classical computing and you get an idea of the power of Q# for expressing the quantum algorithm in a way that abstracts the low level quantum operations.  Again, if you want to start coding, this tutorial guides you in developing the program using a variety of programming environments (with a Python host or with .NET language host and with Visual Studio, Visual Studio Code).  If you are not a coder, you don’t have to install the QDK.  Just follow along the tutorial to get a sense of the nature of quantum computing with Q#.  
+The second Getting Started guide, [Grover’s search](xref:microsoft.quantum.quickstarts.search), offers an example of a Q# program get an idea of the power of Q# for expressing the quantum algorithm in a way that abstracts the low level quantum operations.  Again, if you want to start coding, this guides you in developing the program using a variety of programming environments (with a Python host or with .NET language host and with Visual Studio, Visual Studio Code).  If you are not a coder, you don’t have to install the QDK.  Just follow along the to get a sense of the nature of quantum computing with Q#.  
```

Comment:
```suggestion
The second Quickstart, [Grover’s search](xref:microsoft.quantum.quickstarts.search), offers an example of a Q# program demonstrating the power of Q# for expressing quantum algorithms in a way that abstracts the low level quantum operations.  Again, if you want to start coding, this guides you in developing a quantum application using a variety of programming environments (with a Python or .NET language host and with Visual Studio, Visual Studio Code).  If you are not a coder, you don’t have to install the QDK.  Just follow along the to get a sense of the nature of quantum computing with Q#.  
```

Reply:
I feel like there might be an over-correction here with respect to the "if you're not a coder..." or "if you're not ready to start coding...". These are still technical documentation pages for an SDK. We are still positioning the QDK around _building_ quantum applications. I'm worried that this emphasis might dissuade folks from even trying to install and play around with Q#. For Bell I think this is fine because we walk through fundamental concepts, I'm not sure it's as powerful for Grover's. What are they learning if they're not running the program that they haven't already learned from reading the welcome page & the 5 questions?